lib/apiservers/service/restapi/handlers/vch_get.go

```diff
@@ -246,7 +246,10 @@ func vchToModel(op trace.Operation, vch *vm.VirtualMachine, d *data.Data, execut
 	}
 	model.Runtime.PowerState = string(powerState)
 
-	model.Runtime.DockerHost, model.Runtime.AdminPortal = getAddresses(vchConfig)
+	model.Runtime.DockerHost, model.Runtime.AdminPortal, err = getAddresses(executor, vchConfig)
+	if err != nil {
+		op.Warn("No client IP assigned.")
```

Comment:
How about logging `err` instead of "No client IP assigned." (In this case, it's basically the same message, but if we ever add another error to `getAddresses`, this might be misleading.)

Reply:
Thanks! fixed